float64 p1
float64 p2
float64 p3
float64 p4
float64 p5
float64 p6
float64 T
---
bool exec